### DISCUSSION TITLE
Environment/Leaf Icon

### DISCUSSION THREAD

**Discussion by hisnameisjimmy** [tailwindlabs/heroicons#444] [Ideas]
Something to represent the environment would be great! This could perhaps include a recycle symbol.

Example:
<img width="234" alt="CleanShot 2021-03-13 at 13 13 14@2x" src="https://user-images.githubusercontent.com/740373/111044482-43f91680-8406-11eb-8cf2-8c6b31b37923.png">

**Comment by austriker27** [comment]
Yes please! Recycling and the leaf would be amazing!

**Reply by JKingma** [reply]
+1
